pkgs/applications/version-management/rapidsvn/default.nix

```diff
@@ -18,12 +18,27 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./fix-build.patch
+    # Python 3 compatibility patches
+    (fetchpatch {
+      url = "https://github.com/RapidSVN/RapidSVN/pull/44/commits/2e26fd5d6a413d6c3a055c17ac4840b95d1537e9.patch";
+      hash = "sha256-8acABzscgZh1bfAt35KHfU+nfaiO7P1b+lh34Bj0REI=";
+    })
+    (fetchpatch {
+      url = "https://github.com/RapidSVN/RapidSVN/pull/44/commits/92927af764f92b3731333ed3dba637f98611167a.patch";
+      hash = "sha256-4PdShGcfFwxjdI3ygbnKFAa8l9dGERq/xSl54WisgKM=";
+    })
+    (fetchpatch {
+      url = "https://github.com/RapidSVN/RapidSVN/pull/44/commits/3e375f11d94cb8faddb8b7417354a9fb51f304ec.patch";
+      hash = "sha256-BUpCMEH7jctOLtJktDUE52bxexfLemLItZ0IgdAnq9g=";
+    })
   ];
 
   meta = {
     description = "Multi-platform GUI front-end for the Subversion revision system";
     homepage = "http://rapidsvn.tigris.org/";
     license = lib.licenses.gpl3Plus;
     maintainers = [ lib.maintainers.viric ];
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

Comment:
```suggestion
    platforms = lib.platforms.unix;
```

Reply:
I can switch to that but I used this `meta.platforms` to be coherent with Subversion

https://github.com/NixOS/nixpkgs/blob/12697a1621da942e4b12f864b3bdd57d9ddcbea0/pkgs/applications/version-management/subversion/default.nix#L111

What's best?

Reply:
In general unless we have a strong reason to not do so (like, something that is really designed only to run in Linux + Darwin), it is better to set it to either `platforms.all` (may be to broadly, this includes Windows) or `platforms.unix` (safer-ish because it only allows POSIX systems).

The dependencies itself will be taken care by the dependencies (if any dependency doesn't build in the platform, this will show as the issue of the build failure). This is also good because when someone want to make this package work, in say, FreeBSD for example, they will need to focus in making Subversion work and not this package.

Reply:
Thanks for the explanations!